nucypher/cli/literature.py

```diff
@@ -55,7 +55,7 @@
 * Ursula Node Operator Notice *
 -------------------------------
 
-By agreeing to stake {str(value)} ({str(value.to_nunits())} NuNits):
+By agreeing to stake {tokens} ({nunits} NuNits):
```

Comment:
I think I may have preferred the previous use of `value`. I like having the one value passed, and the conversion to NuNits being automatically performed. I'm interested in your reasoning for the change...?

Reply:
@derekpierre - Raising a good question here... We need a solid guideline for inclusion in `literature.py` (see #2010).  

Primary motivator here for me is to keep python code from appearing in strings and to keep them as pure string templates. 

I think leaving the string-expressed-logic is actually a potential vector for future regression.

Reply:
Ah - fair. Maybe to explicitly keep the relation you can rename `nunits` in the string template to `token_nunits`?

Reply:
> Primary motivator here for me is to keep python code from appearing in strings and to keep them as pure string templates.

Why is this a goal?

> I think leaving the string-expressed-logic is actually a potential vector for future regression.

Also curious to hear your reasoning here. In fact, I think current approach in this particular case is more prone to errors.